modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/exportimport/data/handler/DDMStructureStagedModelDataHandler.java

```diff
@@ -750,6 +751,27 @@ private boolean _isPreloadedStructure(
 		return false;
 	}
 
+	private void _updateDDMFormFieldsDDMStructureIds(
+		DDMForm ddmForm, Map<Long, Long> structureIds) {
+
+		List<DDMFormField> ddmFormFields = ddmForm.getDDMFormFields();
+
+		Stream<DDMFormField> stream = ddmFormFields.stream();
+
+		stream.filter(
+			ddmFormField -> ddmFormField.hasProperty("ddmStructureId")
+		).forEach(
```

Comment:
@mbowerman based on this [message](https://liferay.slack.com/archives/CL69D8RUG/p1649806046880829) from Brian, we should avoid using Stream

Reply:
@carolmariaabb Done